### DISCUSSION TITLE
Security Manager - Review Logs

### DISCUSSION THREAD

**Discussion by brandenwagner** [community/community#15419] [Code Security]
It would be nice if a security manager had access to the audit logs. Currently only owners have access, and it doesn't make sense to give everyone owner access just to read audit logs.

**Comment by jenschelkopf** [comment]
Hi @brandenwagner, I posted a response in [this discussion](https://github.com/github/feedback/discussions/16060). Thanks for the feedback!

**Comment by kellyarwine** [comment]
👋 Hi, @brandenwagner.  Thanks again for the feedback.  I'm the product manager for the Security Manager role and I'm interested in understanding more about your security manager needs as we we work to improve that functionality.  If you have 30 minutes, I'd be open to chatting.  You can set up time with me [here](https://calendly.com/kellyarwine/github-product-interview).
